tests/unit/amo/components/TestSearchResult.js

```diff
@@ -39,93 +18,104 @@ describe('<SearchResult />', () => {
     name: 'A search result',
     slug: 'a-search-result',
   };
-  const root = renderResult(result);
+
+  function render({ addon = baseAddon, lang = 'en-GB', ...props } = {}) {
+    return shallow(
+      <SearchResultBase
```

Comment:
I don't think you need to use the base component. All I see is a `translate()` wrapper so you could use the unwrapping approach above and test the final component directly.